pybossa/jobs.py

```diff
@@ -1338,13 +1339,18 @@ def check_and_send_task_notifications(project_id, conn=None):
 
     update_reminder = False
     if n_remaining_tasks > target_remaining and email_already_sent:
-        current_app.logger.info('incomplete tasks increase over target remaining, reset reminder to active')
+        current_app.logger.info(u'Project {}, the number of incomplete tasks: {} \
+                                exceeds target remaining: {}, reset Sent as True'
```

Comment:
Should this say "reset update_reminder as True"? Or rather, "resetting the send notification flag to True"?

Reply:
will rephrase to "resetting the send notification flag to True"